test/exposed/EverlongAdminExposed.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.20;
+
+import { EverlongBase } from "../../contracts/internal/EverlongBase.sol";
+import { EverlongAdmin } from "../../contracts/internal/EverlongAdmin.sol";
+import { console2 } from "forge-std/console2.sol";
```

Comment:
Since EverlongAdmin is abstract, and I kind of like the pattern of using these exposed contracts within the harnesses, I would personally opt to keep it although it's not doing much....

Will do import reordering 👍

Reply:
Honestly, I love the idea of importing `console` in tests. It's so annoying to constantly have to rewrite it. We should also do the same with `Lib` for `toString`.

 I personally like the alias:

```
import { console2 as console } from "forge-std/console2.sol"
```

since it makes the Vim `<CTRL>a` and `<CTRL>x` hotkeys MUCH more useful when debugging with statements like:

```
console.log("test: 1");
blah();
console.log("test: 2");
foobarbaz();
console.log("test: 3");
```

Reply:
oooo love that very nice